src/rippled/lib/rippled.js

```diff
@@ -74,6 +74,70 @@ const getLedger = (rippledSocket, parameters) => {
   })
 }
 
+// get ledger_entry
+const getLedgerEntry = (rippledSocket, parameters) => {
```

Comment:
This (the `parameters` field) seems unnecessarily complex for this - we can add more parameters later.

Reply:
All of the other functions in this file really only use parameters as-needed, which is why I'm suggesting we do the same here. Nothing here needs to be backwards-compatible, so we can add that later if needed.

Reply:
All of the other functions in this file really only use parameters as-needed, which is why I'm suggesting we do the same here. Nothing here needs to be backwards-compatible, so we can add that later if needed, since it'll be clearer for now.